### ISSUE TITLE
Unable to Read lkml files with filters element

### ISSUE THREAD

**Issue by brunoffaustino** [joshtemple/lkml#36]
When reading a lkml file that has something like the following:

```
  measure: min_example {
    label: "example_label"
    description: "Example Description"
    type: min
    sql:  ${TABLE}.n_example;;
    filters: [n_example: ">0"]
  }
```
We get this error:
```
Traceback (most recent call last):
  File "main.py", line 19, in searching_all_files
    data = lkml.load(stream)
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/__init__.py", line 33, in load
    result = parser.parse()
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 153, in parse
    return self.parse_expression()
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 89, in wrapper
    result = method(self, *args, **kwargs)
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 250, in parse_expression
    block = self.parse_block()
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 89, in wrapper
    result = method(self, *args, **kwargs)
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 322, in parse_block
    expression = self.parse_expression()
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 89, in wrapper
    result = method(self, *args, **kwargs)
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 250, in parse_expression
    block = self.parse_block()
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 89, in wrapper
    result = method(self, *args, **kwargs)
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 322, in parse_block
    expression = self.parse_expression()
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 89, in wrapper
    result = method(self, *args, **kwargs)
  File "/Users/user/.local/share/virtualenvs/example/lib/python3.8/site-packages/lkml/parser.py", line 266, in parse_expression
    raise SyntaxError(
SyntaxError: Unable to find a matching expression for ':' on line 105
```

**Comment by joshtemple** [comment]
Hi @brunoffaustino! Yes, this is an issue with the new Looker filter syntax. I'm hoping to have a more comprehensive fix for this soon, but in the meantime there are a few options. You can use the fix in #32 or there's an open PR (#34) that you could use as a fork.

For now, I'm closing this issue as a duplicate of #32

**Comment by brunoffaustino** [comment]
Hi @joshtemple Thank you. Sorry i didn't see the correlation between both of these issues opened. 

Thank you for looking into it.
Keep up the good work! 💪
